components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/SelectAcrFromFunction.java

```diff
@@ -53,7 +53,7 @@ public String evaluate(JsAuthenticationContext context, Object possibleOutcomesO
         return null;
     }
 
-    private String[] extractPossibleOutcomes(JsAuthenticationContext context, Object possibleOutcomesObj) {
+    private String[] extractPossibleOutcomes(JsBaseAuthenticationContext context, Object possibleOutcomesObj) {
```

Comment:
same as the above

Reply:
Addressed with f00358d0066bbf13b65c87f4867e2ad5052761fc